application/views/invoice_templates/pdf/InvoicePlane - paid.php

```diff
@@ -104,15 +104,15 @@
         <table>
             <tr>
                 <td><?php echo trans('invoice_date') . ':'; ?></td>
-                <td><?php echo date_from_mysql($invoice->invoice_date_created, true); ?></td>
+                <td><?php echo date_from_mysql(htmlsc($invoice->invoice_date_created), true); ?></td>
             </tr>
             <tr>
-                <td><?php echo trans('due_date') . ': '; ?></td>
-                <td><?php echo date_from_mysql($invoice->invoice_date_due, true); ?></td>
+                <td class="text-red"><?php echo trans('due_date') . ': '; ?></td>
+                <td class="text-red"><?php echo date_from_mysql(htmlsc($invoice->invoice_date_due), true); ?></td>
             </tr>
             <tr>
-                <td class="text-green"><?php echo trans('amount_due') . ': '; ?></td>
-                <td class="text-green"><?php echo format_currency($invoice->invoice_balance); ?></td>
+                <td class="text-red"><?php echo trans('amount_due') . ': '; ?></td>
+                <td class="text-red"><?php echo format_currency(htmlsc($invoice->invoice_balance)); ?></td>
```

Comment:
Are the changes maybe a mistake?

Reply:
fixed and checked. There were a couple of more misshaps, fixed them as well